### PR TITLE
Release commit for v2.13.0

### PR DESCRIPTION
**Pull request by vastin**
*Issue #, if available:*
Release v2.13.0

*Description of changes:*
Update version and changelog.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -5,6 +5,16 @@ CHANGELOG
 Unreleased
 ==========
 
+2.13.0
+==========
+* bugfix: Fix passing multiple values in testenv.passenv in tox.ini `https://github.com/aws/aws-xray-sdk-python/pull/399`
+* improvement: Pin flask < 3.x for flask sqlalchemy tests `https://github.com/aws/aws-xray-sdk-python/pull/412`
+* improvement: Bump werkzeug from 2.2.3 to 3.0.1 in /sample-apps/flask `https://github.com/aws/aws-xray-sdk-python/pull/413`
+* improvement: Enable testing on Python 3.12 `https://github.com/aws/aws-xray-sdk-python/pull/400`
```
Comment (atshaw43) [inline CHANGELOG.rst:13]: We don't need to mentioned internal facing changes such as supporting new tests.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -5,6 +5,16 @@ CHANGELOG
...
+* bugfix: Fix typo in docs `https://github.com/aws/aws-xray-sdk-python/pull/419`
```
Comment (jj22ee) [inline CHANGELOG.rst:14]: Instead of `bugfix`, can we label as `typo` or `improvement`?

File: CHANGELOG.rst (hunk 1 above)
Comment (vastin) [reply CHANGELOG.rst]: fixed. ty

File: CHANGELOG.rst (hunk 2 above)
Comment (vastin) [reply CHANGELOG.rst]: Fixed. ty.

**Review by jj22ee — APPROVED** [review]

**Review by atshaw43 — APPROVED** [review]

**Pull request merged**